src/utils/templates/component.jsx

```diff
@@ -1,5 +1,7 @@
-export default (name) => {
-	const nameLcase = name.replace(/^[A-Z]/, (firstLetter) => firstLetter.toLowerCase());
+export default name => {
```

Comment:
Let's file a follow up ticket to remove or update these templates

Reply:
Filed